composer.json

```diff
@@ -14,16 +14,17 @@
         }
     },
     "require": {
-        "php": "^5.5 || ^7.0",
-        "zendframework/zend-loader": "^2.5",
-        "zendframework/zend-stdlib": "^2.5 || ^3.0",
-        "zendframework/zend-uri": "^2.5",
-        "zendframework/zend-validator": "^2.5"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-loader": "^2.5.1",
```

Comment:
Updating minimum versions for those packages is unrelated to either of 1) phpunit 2) php version bump 3) coding style fixes 4) travis, unless they were failing  with new test setup.

Hence it is outside of the scope of this PR.

Anyway, disregard that.